http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -345,6 +324,9 @@ object Boot extends IOApp {
       samDao <- buildHttpClient(sslContext, proxyResolver.resolveHttp4s, Some("leo_sam_client"), true).map(client =>
         HttpSamDAO[F](client, httpSamDaoConfig, petTokenCache)
       )
+      cromwellDao <- buildHttpClient(sslContext, proxyResolver.resolveHttp4s, Some("leo_cromwell_client"), true).map(
```

Comment:
WRT the above comment, can pass `false` here to avoid retrying in the http4s client.